extensions/src/ACRE2Mumble/MumbleClient.cpp

```diff
@@ -71,12 +71,13 @@ acre::Result CMumbleClient::start(const acre::id_t id_) {
 }
 
 bool CMumbleClient::getVAD() {
-    transmission_mode_t transmitMode;
+    mumble_transmission_mode_t transmitMode;
     const mumble_error_t err = mumAPI.getLocalUserTransmissionMode(pluginID, &transmitMode);
     if (err != ErrorCode::EC_OK) {
         return false;
     }
 
+	// TODO: Is this enum defined somewhere in ACRE?
     return transmitMode == TransmissionMode::TM_VOICE_ACTIVATION;
```

Comment:
:point_up:

Reply:
Mumble includes from that time: https://github.com/IDI-Systems/acre2/blob/a0572a4dd4fe871b8b896e544300605f4cd0d87c/extensions/src/ACRE2Mumble/mumble_includes/PluginComponents.h#L55-L62